README.md

```diff
@@ -84,7 +84,7 @@ Each contains a set of chunks: a common base, and a language, and includes Docke
 
 These images are no longer being published:
 
-- gitpod/workspace-python-3.6 (please use [`gitpod/workspace-python-3.7`](https://hub.docker.com/r/gitpod/workspace-python-3.7) instead)
+- gitpod/workspace-python-3.9 or older (please use [`gitpod/workspace-python-3.11`](https://hub.docker.com/r/gitpod/workspace-python-3.11) instead)
```

Comment:
What defines when versions will be retired?

Reply:
Thank you for asking. I will respond to the email that I just received from you.